src/processTargets/index.ts

```diff
@@ -142,6 +155,45 @@ function processRangeTarget(
   );
 }
 
+function processColumnTarget(
+  target: RangeTarget,
+  anchorTarget: TypedSelection,
+  activeTarget: TypedSelection,
+  isForward: boolean
+) {
+  const anchorLine = targetToLineLimitPosition(
+    anchorTarget,
+    isForward,
+    !target.excludeAnchor
+  );
+  const activeLine = targetToLineLimitPosition(
+    activeTarget,
+    !isForward,
+    !target.excludeActive
+  );
+  const startLine = isForward ? anchorLine : activeLine;
+  const endLine = isForward ? activeLine : anchorLine;
+  const anchorSelection = anchorTarget.selection.selection;
+  const startCharacter = anchorSelection.start.character;
+  const endCharacter = anchorSelection.end.character;
+  const results: TypedSelection[] = [];
+
+  for (let i = startLine; i <= endLine; ++i) {
```

Comment:
So I think to be the most consistent with what the user said, we should probably add the selections in order from anchor to active, rather than start to end.  Going to be a bit confusing when they try to sort a backwards list of selections, but it is consistent with the kind of thing we do elsewhere

Reply:
done